vllm/model_executor/models/registry.py

```diff
@@ -180,10 +180,10 @@
     "LlavaNextForConditionalGeneration": ("llava_next", "LlavaNextForConditionalGeneration"),  # noqa: E501
     "Phi3VForCausalLM": ("phi3v", "Phi3VForCausalLM"),
     "Qwen2VLForConditionalGeneration": ("qwen2_vl", "Qwen2VLForConditionalGeneration"),  # noqa: E501
-    # Technically PrithviGeoSpatialMAE is a model that works on images, both in
+    # Technically Terratoch models work on images, both in
     # input and output. I am adding it here because it piggy-backs on embedding
     # models for the time being.
-    "PrithviGeoSpatialMAE": ("prithvi_geospatial_mae", "PrithviGeoSpatialMAE"),
+    "Terratorch": ("terratorch", "Terratorch"),
```

Comment:
Are there any old model repos that still need to be supported?
If yes you could leave the alias here:
```
"PrithviGeoSpatialMAE": ("terratorch", "Terratorch"),
```